README.rst

```diff
@@ -87,12 +87,13 @@ Get historical data for specific stations, using Python:
 
 .. code-block:: python
 
-    from wetterdienst import DWDObservationData, Parameter, PeriodType, TimeResolution
+    from wetterdienst.dwd.observations import DWDObservationData, DWDObsParameterSet,
+        DWDObsPeriodType, DWDObsTimeResolution
```

Comment:
Can we use long names here? So, `DWDObservationParameterSet`, `DWDObservationPeriodType` and `DWDObservationTimeResolution`.

Reply:
Maybe also strip off the `Set` suffix and call it `DWDObservationParameter` again. Saying that (as we are already renaming stuff right now), I would also like `DWDObservationPeriod`, stripping off the `Type` suffix. If we also want to adjust the third guy, maybe let's say `DWDObservationResolution`?

All at once:
- `DWDObservationParameter`
- `DWDObservationPeriod`
- `DWDObservationResolution`

Would that resonate with you?

Reply:
Stripping off Set is not recommended as it refers to a set of parameters (for example climate summary consists of multiple parameters). I did this change to accomodate for other weather services, where we'll likely only of separate parameters available instead of those "sets".

About the abbrevation: I did this to get an as short as possible name, as forecast parameters are getting really long e.g.

PROBABILITY_SUNSHINE_DURATION_RELATIVE_GT_60_PCT_LAST_24H

which is why I first shortened the forecast parameter accessor to DWDFcstParameter.

But maybe you are right and this is just like a trop in the bucket...